include/fmt/format-inl.h

```diff
@@ -216,6 +219,10 @@ template <typename Char> FMT_FUNC Char decimal_point_impl(locale_ref loc) {
 }  // namespace internal
 #else
 template <typename Char>
+FMT_FUNC std::string internal::grouping_impl(locale_ref) {
+  return "\03";
```

Comment:
Does this actually work? I think this will call the `std::string` ctor taking `const char*` and result in an empty string.

Reply:
Nevermind, this is a single octal escape =).